advisor-schema/src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,95 @@
+--CREATE SCHEMA IF NOT EXISTS advisory_schema;
+--USE SCHEMA 'advisory_schema';
+
+CREATE TYPE user_type AS ENUM (
+    'applicant',
+    'advisor'
+    );
+
+CREATE TYPE advisors_role AS ENUM (
+    'associate',
+    'partner',
+    'senior'
+    );
+
+CREATE TYPE phone_number_type AS ENUM (
+    'home',
+    'work',
+    'mobile'
+    );
+
+CREATE TYPE application_status AS ENUM (
+    'new',
+    'assigned',
+    'on_hold',
+    'approved',
+    'canceled',
+    'declined'
+    );
+
+CREATE TABLE IF NOT EXISTS users
+(
+    id       BIGSERIAL CONSTRAINT users_PK PRIMARY KEY,
+    username VARCHAR(255) NOT NULL CONSTRAINT users_username_UK UNIQUE,
+    email    VARCHAR(255) NOT NULL CONSTRAINT users_email_UK UNIQUE,
+    "type"   user_type    NOT NULL,
+    version                    BIGINT                   NOT NULL DEFAULT 0,
+    CONSTRAINT email_format_check CHECK (email ~* '^[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}$')
+);
+
+CREATE TABLE IF NOT EXISTS advisors
+(
+    id     BIGINT CONSTRAINT advisors_PK PRIMARY KEY,
+    "role" advisors_role NOT NULL,
```

Comment:
why do you use brackets for some field`s names?
it works without brackets

Reply:
To escape a reserved word, such as a column or table name. Read about that on official documentation